extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -114,4 +114,13 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 
 		return [group, changeTargetPlatformAction];
 	}
+
+	/** Adds the list of files and directories to the project, and saves the project file
+	* @param projectFile The Uri of the project file
+	* @param list list of uris of files and folders to add. Files and folders must already exist. Files and folders must already exist. No files or folders will be added if any do not exist.
+	*/
+	async addToProject(projectFile: vscode.Uri, list: vscode.Uri[]): Promise<void> {
+		const project = await Project.openProject(projectFile.fsPath);
+		await project.addToProject(list);
```

Comment:
is the data workspace extension going to reload the project (so the changes are loaded in the the UI) after this is done, or should that be getting done here?

Reply:
@kisantia might be able to answer this better. She worked on these changes on her PR (the change comes in from merging from 'main' branch).

Reply:
good catch. I thought it was getting refreshed, but that's just because I was changing views when I was testing this. Will send a PR to add refresh

Reply:
actually, this might not be possible. Refresh needs the workspaceTreeItem to notify that the tree changed, but this can get called from another extension instead of from right clicking on the project tree. I think it's fair for the other extension calling addToProject() to just call the data-workspace refresh command.